src/views/Pipeline.vue

```diff
@@ -554,11 +555,23 @@ export default defineComponent({
 
       await this.store.dispatch('user/updatePinnedJobs', { pinnedJobs: [...pinnedJobs] });
       this.updateSelectedPinnedJob(enumId)
+    },
+    updateCurrentJob() {
+      if(!this.getCurrentJob) {
+        this.currentJob = ''
+      } else {
+        this.viewJobConfiguration(this.getCurrentJob);
+      }
     }
   },
   created() {
     this.getPendingJobs();
     this.store.dispatch('user/getPinnedJobs');
+    emitter.on('updatePendingJobs', this.getPendingJobs);
```

Comment:
I think we should subscribe to a single emitter and handle everything in it

Reply:
Sir, Updated code.